iphone/Classes/KrollContext.m

```diff
@@ -1141,6 +1143,109 @@ - (int)forceGarbageCollectNow
   return 0;
 }
 
+// Worker class definition
+TiClassRef TiWorker_class(TiContextRef context)
+{
+  static TiClassRef jsClass;
+  if (!jsClass) {
+    TiClassDefinition definition = kTiClassDefinitionEmpty;
+    definition.className = "Worker";
+    definition.initialize = TiWorker_initialize;
+    definition.finalize = TiWorker_finalize;
+    definition.staticFunctions = TiWorker_staticFunctions;
+    definition.setProperty = TiWorker_setProperty;
```

Comment:
add:
`definition.callAsConstructor = TiWorker_construct;`

Reply:
Should this go here as well? It's referenced in the `TiObjectMakeConstructor` call right now.